tasks/installation.yml

```diff
@@ -27,6 +27,21 @@
   when: bbb_webhooks_enable
   notify: restart bigbluebutton
 
+- name: install bbb api demos
+  apt:
+    name: "{{ bbb_demos }}"
+    state: "{{ bbb_state }}"
+    # Using autoremove because otherwise tomcat8
+    # would stay installed and keep running
+    autoremove: true
```

Comment:
That line is wrong here and should move down, it seems as if I was a bit tired yesterday. I will fix that during the day and would recommend not merging before I fixed that.